shuup/front/apps/registration/templates/shuup/registration/company_register.jinja

```diff
@@ -1,25 +1,9 @@
 {% extends "shuup/front/base.jinja" %}
 {% from "shuup/front/macros/general.jinja" import render_field %}
+{% from "shuup/front/macros/checkout.jinja" import render_address_form %}
```

Comment:
Why `render_address_form` is imported and in line 57 you create macro with the same name?